.github/workflows/main.yml

```diff
@@ -165,12 +165,41 @@ jobs:
       #       'halt\n' \
       #     | docker exec -i $CONTAINER iris session iris -UUSER
       #     docker stop $CONTAINER
+      - name: Clean install v0.7.x and test migration to latest.
+        timeout-minutes: 15
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          curl http://localhost:52773/registry/packages/-/all | jq
+          curl http://localhost:52773/registry/packages/zpm/ | jq
```

Comment:
Are we assuming localhost:52773 is running the correct kit here?

Reply:
Huh... I got this from the above section, but it makes sense neither place. Good catch.

Reply:
Actually - on further review, there's a binding of port 52773 on the runner to the same port on the registry container, so this is OK. (I guess it would fail early if there's something wrong with the registry?)